Homework1: Egor Brel

DwarfCommunist opened the pull request:
### PR checklist
Please make sure you've passed PR checklist
- [x] my PR passes CI checks (flake8 and pycodestyle)
- [x] my PR named according to guidelines.
- [ ] my PR contains complete tasks descriptions for all tasks
- [ ] my PR made from the fork feature branch to maxim homework# branch.
- [x] my PR contains only code that solves appropriate homework tasks. Redundant 
files (configs, etc. ) are put to .gitignore

Commit:
Initial commit. Simple programs.

Commit:
flake8 and pycodestyle requirements support

belovmd reviewed (commented):
Thank you for your work. 
Please address next paragraphs from review notes [Review Notes](https://github.com/belovmd/it-academy-autumn/blob/master/hw_review/homework1.md):
- 2

Commit:
Add desciption to tasks

Add task titles and descriptions at the begining of file, in order to
satisfy hm_review document (point number 2).

DwarfCommunist commented:
Hi @belovmd.
Thanks for the review. I've added titles and task description at the beginning of each file.

belovmd reviewed (approved):
+1
